CONTRIBUTE.md

```diff
@@ -0,0 +1,102 @@
+# Contributing Guidelines for firecamp.io
+
+Thank you for considering contributing to firecamp.io! To ensure smooth collaboration and maintain a consistent workflow, please follow the guidelines below.
+
+## Commit Guidelines
+
+- We encourage using either the learna or commitzen format for your commit messages.
+- Keep your commit messages concise, clear, and descriptive.
+- Make sure to include a brief summary of the changes made in the commit.
+
+Example Commit Message:
+
+```
+feat: Add new authentication module
+
+```
+
+## Fork and Pull Request Workflow
+
+1. Fork the firecamp.io repository to your GitHub account.
+2. Create a new branch for your changes. Use a descriptive and meaningful name for the branch that reflects the purpose of your contribution.
+3. Make your changes in the branch, following the project's coding style and guidelines.
+4. Commit your changes with appropriate messages, adhering to the commit guidelines mentioned above.
+5. Push your branch to your forked repository.
+6. Create a pull request (PR) from your branch to the main firecamp.io repository.
+7. Provide a clear and concise title for your pull request, summarizing the changes made.
+8. In the PR description, provide a detailed explanation of the problem you solved or the feature you added. If applicable, include relevant screenshots or a video showcasing the changes.
+9. Tag your PR with one or more relevant labels from the following options: `graphql`, `ui`, `documentation`, `rest`, `backend`, or any other appropriate labels.
+
+Example branch name
+```
+auth-module
+```
+
+Example PR title
+```
+feat: Add new authentication module
+
+```
+
+Example PR description
+
+```
+## Description
+This pull request adds a new authentication module to improve user security and access control.
+
+## Changes Made
+- Implemented JWT-based authentication
+- Added user registration and login endpoints
+- Enhanced password encryption and storage
+
+## Screenshots
+![Screenshot 1](/images/screenshot1.png)
+![Screenshot 2](/images/screenshot2.png)
+
+Fixes #123
+
+```
+
+## Issue Creation
+
+1. Before creating a new issue, search the issue tracker to ensure it doesn't already exist.
+2. Create a new issue if it hasn't been reported yet.
+3. Give the issue a descriptive title, summarizing the problem or feature request concisely.
+4. In the issue description, provide clear steps to reproduce the problem or describe the desired feature in detail. If relevant, include a screenshot or a video to aid understanding.
+5. Tag the issue with appropriate labels to categorize it correctly.
+
+Example issue title
+
+```
+Bug: Invalid characters in username field
+
+```
+
+Example issue description
+```
+## Issue Description
+When a user enters special characters in the username field during registration, the application throws an error and fails to save the user.
```

Comment:
would be fine to add the issue templates too.

Here is the bug issue template, you can change it inorder to improve.
https://github.com/FirecampDev/Firecamp/blob/main/.github/ISSUE_TEMPLATE/bug_report.md